src/box/lua/load_cfg.lua

```diff
@@ -699,6 +703,7 @@ local dynamic_cfg_order = {
     replication_synchro_queue_max_size = 150,
     replication_connect_timeout = 150,
     replication_connect_quorum  = 150,
+    replication_linearizable_quorum = 150,
```

Comment:
Shouldn't we set this option after `replication_synchro_quorum` ?

Otherwise we might end up with temporarily broken quorum values, when the user sets `synchro` and `linearizable` quorum together.

Reply:
Didn't manage to catch any broken quorum, when linearizable quorum had 150 weight, but anyway changed it to 175, better safe than sorry